trusted.ci.jenkins.io.tf

```diff
@@ -69,6 +69,18 @@ resource "azurerm_role_assignment" "trusted_ci_jenkins_io_allow_packer" {
   principal_id         = azuread_service_principal.trusted_ci_jenkins_io.id
 }
 
+# DNS
+resource "azurerm_private_dns_zone" "trusted" {
+  name                = "trusted.ci.jenkins.io"
+  resource_group_name = data.azurerm_resource_group.trusted.name
+}
+resource "azurerm_private_dns_zone_virtual_network_link" "trusted" {
+  name                  = "trustedDNS"
```

Comment:
```suggestion
resource "azurerm_private_dns_zone_virtual_network_link" "trustedvnet_to_trusted_ci_jenkins_io" {
  name                  = "trusted-vnet-to-trusted.ci.jenkins.io"
```

Reply:
this is overly complex and hard to read. you will be looking at this from the trusted.ci.jenkins.io zone so a short name is appropriate here. tbh not really sure why a name is even needed the vnet is already displayed.

I would just repeat the vnet name or something simple:
![image](https://user-images.githubusercontent.com/21194782/236187040-f6cac007-2485-4f07-bdbd-ab9def560bd8.png)

Reply:
```suggestion
resource "azurerm_private_dns_zone_virtual_network_link" "trusted" {
  name                  = "trusted-vnet"
```

Reply:
thanks @lemeurherve did this here : https://github.com/jenkins-infra/azure/pull/341